srcpkgs/vosk-api/template

```diff
@@ -0,0 +1,79 @@
+# Template file for 'vosk-api'
+pkgname=vosk-api
+version=0.3.43
+revision=1
+create_wrksrc=yes
+build_wrksrc="vosk-api-${version}/src"
+_openfst_commit=7dfd808194105162f20084bb4d8e4ee4b65266d5
+_kaldi_commit=6417ac1dece94783e80dfbac0148604685d27579
+build_style=gnu-makefile
+make_use_env=yes
+make_build_args="KALDI_ROOT=../../kaldi-${_kaldi_commit}
+	OPENFST_ROOT=../../kaldi-${_kaldi_commit}/tools/openfst
+	OPENBLAS_ROOT=../../kaldi-${_kaldi_commit}/tools/OpenBLAS/install"
+makedepends="blas-devel lapack-devel openblas-devel libgomp-devel openfst-devel"
+short_desc="Offline speech recognition toolkit"
+maintainer="John Gebbie <me@johngebbie.com>"
+license="Apache-2.0"
+homepage="https://alphacephei.com/vosk"
+distfiles="https://github.com/alphacep/vosk-api/archive/v${version}/vosk-api-${version}.tar.gz
+https://github.com/alphacep/kaldi/archive/${_kaldi_commit}.tar.gz
+https://alphacephei.com/vosk/models/vosk-model-small-en-us-0.15.zip"
+checksum="50fa82bfbfe3fc5be868237c84ac180eb55da0b65b1803bf022899673afda024
+4e5c08c9c61bbc995e0620321feb0973bb0321442f1bd8f08b47497f55efadda
+30f26242c4eb449f948e42cb302dd7a686cb29a3423a8367f99ff41780942498"
+shlib_provides="libvosk.so"
+patch_args="-d kaldi-${_kaldi_commit} -Np1"
+
+if [ "$XBPS_TARGET_LIBC" = musl ]; then
+	makedepends+=" libexecinfo-devel"
+fi
+
+post_extract() {
+	# Don't need Fortran to C, we have lapack not clapack.
+	cd "${wrksrc}/kaldi-${_kaldi_commit}/src"
+	vsed -i configure -e 's/ -l:libf2c\.a//'
+	cd "${wrksrc}/vosk-api-${version}/src"
+	vsed -i Makefile -e '/libblas\.a \\/ {N; s/ \\.*//}'
+	vsed -i Makefile -e 's/ -lf2c//'
```

Comment:
I think this should also be a patch.

Reply:
I didn't make this into patches but I improved the second last vsed, making it explicit and less fragile.
I can still make it into patches if you'd prefer that.

Reply:
I have now made them patches and put links to my pull requests in the template